src/auth/apps.rs

```diff
@@ -0,0 +1,32 @@
+use jsonwebtoken::{encode, Algorithm, EncodingKey, Header};
```

Comment:
Can we move this into `src/auth.rs` directly instead?
```suggestion
use jsonwebtoken::{Algorithm, EncodingKey, Header};
```